src/dmd/expression.d

```diff
@@ -79,6 +79,16 @@ void emplaceExp(T : UnionExp)(T* p, Expression e)
     memcpy(p, cast(void*)e, e.size);
 }
 
+// Return value for `checkModifiable`
+enum Modifiable
+{
+    /// Not modifiable
+    No,
+    /// Modifiable (the type is mutable)
+    Yes,
+    /// Modifiable because it is initialization
+    Initialization,
```

Comment:
Should have all small letters to follow the standard D style.

Reply:
Fixed